pkg/minikube/vmpath/constants.go

```diff
@@ -24,7 +24,7 @@ const (
 	// GuestEphemeralDir is the path where ephemeral data should be stored within the VM
 	GuestEphemeralDir = "/var/tmp/minikube"
 	// GuestPersistentDir is the path where persistent data should be stored within the VM (not tmpfs)
-	GuestPersistentDir = "/var/lib/minikube"
+	GuestPersistentDir = "/usr/local/minikube"
```

Comment:
/usr is on /tmpfs in the current ISO, which means it can't be used for persistent storage without changing it.

Reply:
what would be a better place for it ?